src/test/suite/StartIntegration.test.ts

```diff
@@ -112,6 +156,29 @@ suite('Check can deploy default examples', () => {
 
 });
 
+async function checkConfigMapAvailableForDeployedIntegration() {
+	const describeShell = shelljs.exec(`${await kamel.create().getPath()} describe integration test-java-deploy-with-config-map`);
+	const description: string = describeShell.stdout;
+	const lineReturnAndSpaces: RegExp = /\r?\n|\r|\s/g;
+	expect(description.replace(lineReturnAndSpaces, '')).includes('Configuration:Type:configmapValue:my-configmap');
+}
+
+async function checkPropertyAvailableAvailableForDeployedIntegration() {
+	const describeShell = shelljs.exec(`${await kamel.create().getPath()} describe integration test-java-deploy-with-property`);
+	const description: string = describeShell.stdout;
+	const lineReturnAndSpaces: RegExp = /\r?\n|\r|\s/g;
+	expect(description).includes('propertyKey=my Value');
+	expect(description.replace(lineReturnAndSpaces, '')).includes('Configuration:Type:propertyValue:propertyKey=myValue');
```

Comment:
an even better check would be to evaluate properties in pod context or to create a route using the property and logging, then to check the log of the pod. But it seems to complicate a lot the test, not sure that it is worthy. it can be done anyway in another iteration.